src/calculators/MSRATeamDivisionPointsCalc.ts

```diff
@@ -61,7 +61,7 @@ export type TeamResult = {
  */
 export const pointsByDivision = (resultsData: Results): Map<string, TeamResult[]> => {
   const regattaInfoJson = JSON.parse(resultsData.regattaInfo.json || '{}');
-  console.log(typeof (regattaInfoJson))
+  console.log(typeof regattaInfoJson);
```

Comment:
Can we remove this console.log statement?

Reply:
Thanks, good catch!